src/mono/System.Private.CoreLib/src/ILLink/ILLink.Substitutions.Intrinsics.x86.xml

```diff
@@ -39,6 +39,12 @@
     <type fullname="System.Runtime.Intrinsics.X86.Avx512DQ/X64">
       <method signature="System.Boolean get_IsSupported()" body="stub" value="false" />
     </type>
+    <type fullname="System.Runtime.Intrinsics.X86.Avx10v1">
+      <method signature="System.Boolean get_IsSupported()" body="stub" value="false" />
+    </type>
+    <type fullname="System.Runtime.Intrinsics.X86.Avx10v1/V512">
```

Comment:
I think we're missing `Avx10v1/X64` and `Avx10v1/V512/X64`

Reply:
Added.